internal/hcsoci/hcsdoc_wcow.go

```diff
@@ -61,51 +61,40 @@ func createWindowsContainerDocument(coi *createOptionsInternal) (interface{}, er
 		v2Container.GuestOs = &hcsschema.GuestOs{HostName: coi.Spec.Hostname}
 	}
 
-	if coi.Spec.Windows.Resources != nil {
-		if coi.Spec.Windows.Resources.CPU != nil {
-			if coi.Spec.Windows.Resources.CPU.Count != nil ||
-				coi.Spec.Windows.Resources.CPU.Shares != nil ||
-				coi.Spec.Windows.Resources.CPU.Maximum != nil {
-				v2Container.Processor = &hcsschema.Processor{}
-			}
-			if coi.Spec.Windows.Resources.CPU.Count != nil {
-				cpuCount := *coi.Spec.Windows.Resources.CPU.Count
-				hostCPUCount := uint64(runtime.NumCPU())
-				if cpuCount > hostCPUCount {
```

Comment:
@jhowardmsft - Whats your opinion here? I am under the impression that this validation is extra because the host will do it anyways. I would prefer a failure here over a silent change to runtime behavior.

Reply:
Spoke offline. Will make a fix here but it will be separate PR because its broken in two places unrelated to this change.

Reply:
Will rebase after #530